bank/services/stoneBaas/stoneBaas.go

```diff
@@ -1,14 +1,118 @@
 package stoneBaas
 
 import (
+	"fmt"
 	"html/template"
+	"net/http"
 	"strconv"
 	"strings"
+	"sync"
 
+	"github.com/PMoneda/flow"
+	"github.com/mundipagg/boleto-api/certificate"
 	"github.com/mundipagg/boleto-api/config"
+	"github.com/mundipagg/boleto-api/log"
+	"github.com/mundipagg/boleto-api/metrics"
+	"github.com/mundipagg/boleto-api/models"
 	"github.com/mundipagg/boleto-api/tmpl"
+	"github.com/mundipagg/boleto-api/util"
+	"github.com/mundipagg/boleto-api/validations"
 )
 
+var (
+	onceTransport = &sync.Once{}
+	transportTLS  *http.Transport
+)
+
+type bankStoneBaas struct {
+	validate    *models.Validator
+	log         *log.Log
+	trasportTLS *http.Transport
+}
+
+func New() (bankStoneBaas, error) {
+	var err error
+	b := bankStoneBaas{
+		validate: models.NewValidator(),
+		log:      log.CreateLog(),
+	}
+
+	b.configureMTLS(&err)
+
+	if err != nil {
+		return bankStoneBaas{}, fmt.Errorf("fail on load TLSTransport: %v", err)
+	}
+
+	b.pushRequestValidations()
+
+	return b, nil
+}
+
+func (b bankStoneBaas) ProcessBoleto(boletoRequest *models.BoletoRequest) (models.BoletoResponse, error) {
+	errs := b.ValidateBoleto(boletoRequest)
+
+	if len(errs) > 0 {
+		return models.BoletoResponse{Errors: errs}, nil
+	}
+
+	return b.RegisterBoleto(boletoRequest)
+}
+
+func (b bankStoneBaas) RegisterBoleto(boletoRequest *models.BoletoRequest) (models.BoletoResponse, error) {
```

Comment:
RegisterBoleto é chamado por outra função além do Process? Se não for acho que vale deixar ela privada.
```suggestion
func (b bankStoneBaas) registerBoleto(boletoRequest *models.BoletoRequest) (models.BoletoResponse, error) {
```

Reply:
Acho que não é não, mas vou checar! Se não for, vou deixar ela privada!!

Reply:
É que a interface de bank tá dizendo que precisa ter esse método dispónivel :/ 
![image](https://github.com/mundipagg/boleto-api/assets/92492921/3f8ac4d2-1a22-402d-9d4b-fd132549e9cb)

Na prática, apenas o ProcessBoleto é chamado fora do package para registrar. Acho que temos a opção de mudar a interface, mas daí temos que mudar nas outras integrações tbm. O que acha?

Reply:
Acho uma boa não alterarmos as outras integrações nesse PR 😀

Reply:
Também acho uma boa não alterarmos. Como a mudança na interface implicaria na mudança das outras integrações, vou manter o método público por enquanto

Reply:
Puts, só vi agora, perdão a resposta demorada. Mas concordo com o Bruno também, melhor opção foi deixar sem alterar as outras integrações.